src/test/java/picocli/ArgGroupTest.java

```diff
@@ -4106,24 +4119,87 @@ public OneOrTwo() {
 
         static class GroupTwo {
             @Option(names = { "-2a", "--option-2a" },required=true, defaultValue = "Default 2A", description = "option A of group 2")
-            private String _2a;
+            private String _2a = "Default 2A";
 
             @Option(names = { "-2b", "--option-2b" },required=true, defaultValue = "Default 2B", description = "option B of group 2")
-            private String _2b;
+            private String _2b = "Default 2B";
         }
     }
 
-    @Ignore
+    /**
+     * Tests issue 1409 https://github.com/remkop/picocli/issues/1409
+     * This specific test supplies values to the group one values, leaving
+     * the group two values uninitialized.
+     * <p>
+     * The test verifies that x, 1A, 1B, 2A, and 2B values are correct after
+     * building the command using CommandLine.java and parsing the arguments.
+     * @author remkop, madfoal
+     * @version 1
+     * @params null
+     * @returns null
+     *
+     */
     @Test
     public void testIssue1409() {
-        Issue1409 obj = new Issue1409();
-//        new CommandLine(obj).parseArgs("-x", "ANOTHER_VALUE", "-2a=x", "-2b=z");
+        final Issue1409 obj = new Issue1409();
         new CommandLine(obj).parseArgs("-x", "ANOTHER_VALUE");
-        assertEquals("ANOTHER_VALUE", obj.optXAndGroupOneOrGroupTwo.x);
-        assertEquals(null, obj.optXAndGroupOneOrGroupTwo.oneORtwo.one._1a);
-        assertEquals(null, obj.optXAndGroupOneOrGroupTwo.oneORtwo.one._1b);
-        assertEquals("Default 2A", obj.optXAndGroupOneOrGroupTwo.oneORtwo.two._2a);
-        assertEquals("Default 2B", obj.optXAndGroupOneOrGroupTwo.oneORtwo.two._2b);
+        assertEquals("Default value for X incorrect","ANOTHER_VALUE", obj.optXAndGroupOneOrGroupTwo.x);
+        assertEquals("Default value for _1a incorrect",null, obj.optXAndGroupOneOrGroupTwo.oneORtwo.one._1a);
+        assertEquals("Default value for _1b incorrect",null, obj.optXAndGroupOneOrGroupTwo.oneORtwo.one._1b);
+        assertEquals("Default value for _2a incorrect","Default 2A", obj.optXAndGroupOneOrGroupTwo.oneORtwo.two._2a);
+        assertEquals("Default value for _2b incorrect","Default 2B", obj.optXAndGroupOneOrGroupTwo.oneORtwo.two._2b);
+
+    }
+
+    /**
+     * Tests issue 1409 https://github.com/remkop/picocli/issues/1409
+     * This specific test supplies values to the group one values, leaving
+     * the group two values uninitialized.
+     * <p>
+     * The test verifies that x, 1A, 1B, 2A, and 2B values are correct after
+     * building the command using CommandLine.java and parsing the arguments.
+     * @author madfoal
+     * @version 1
+     * @params null
+     * @returns null
+     *
```

Comment:
Similarly, please remove the `@version, @params, @returns` and the empty last line in the javadoc.

Reply:
Edits made for the three JUnit tests.